tensorflow_app/views/layers_import.py

```diff
@@ -0,0 +1,530 @@
+
+    strides = [int(i) for i in depthwise_conv_op.get_attr('strides')]
+    layer_params['stride_h'] = strides[1]
+    layer_params['stride_w'] = strides[2]
+
+    pad_h, pad_w = get_padding(depthwise_conv_op, kernel_shape, strides)
+    layer_params['pad_h'] = pad_h
+    layer_params['pad_w'] = pad_w
+
+    return jsonLayer('DepthwiseConv', layer_params, get_input_layers(layer_ops), [])
+
+
+def import_pooling2d(layer_ops):
+    pooling2d_op = next((x for x in layer_ops if x.type in ['MaxPool', 'AvgPool']))
+
+    layer_params = {}
+
+    layer_params['padding'] = str(pooling2d_op.get_attr('padding'))
+    layer_params['layer_type'] = '2D'
+
+    # checking type of pooling layer
+    if pooling2d_op.type == 'MaxPool':
+        layer_params['pool'] = 'MAX'
+    elif pooling2d_op.type == 'AvgPool':
+        layer_params['pool'] = 'AVE'
+
+    kernel_shape = [int(i) for i in pooling2d_op.get_attr('ksize')]
+    layer_params['kernel_h'] = kernel_shape[1]
+    layer_params['kernel_w'] = kernel_shape[2]
+
+    strides = [int(i) for i in pooling2d_op.get_attr('strides')]
+    layer_params['stride_h'] = strides[1]
+    layer_params['stride_w'] = strides[2]
+
+    pad_h, pad_w = get_padding(pooling2d_op, kernel_shape, strides)
+    layer_params['pad_h'] = pad_h
+    layer_params['pad_w'] = pad_w
+
+    return jsonLayer('Pooling', layer_params, get_input_layers(layer_ops), [])
+
+
+def import_pooling3d(layer_ops):
+    pooling3d_op = next((x for x in layer_ops if x.type in ['MaxPool3D', 'AvgPool3D']))
+    layer_params = {}
+
+    layer_params['padding'] = str(pooling3d_op.get_attr('padding'))
+    layer_params['layer_type'] = '3D'
+
+    # checking type of pooling layer
+    if pooling3d_op.type == 'MaxPool':
+        layer_params['pool'] = 'MAX'
+    elif pooling3d_op.type == 'AvgPool':
+        layer_params['pool'] = 'AVE'
+
+    kernel_shape = [int(i) for i in pooling3d_op.get_attr('ksize')]
+    layer_params['kernel_d'] = kernel_shape[1]
+    layer_params['kernel_h'] = kernel_shape[2]
+    layer_params['kernel_w'] = kernel_shape[3]
+
+    strides = [int(i) for i in pooling3d_op.get_attr('strides')]
+    layer_params['stride_d'] = strides[1]
+    layer_params['stride_h'] = strides[2]
+    layer_params['stride_w'] = strides[3]
+
+    pad_d, pad_h, pad_w = get_padding(pooling3d_op, kernel_shape, strides)
+    layer_params['pad_d'] = pad_d
+    layer_params['pad_h'] = pad_h
+    layer_params['pad_w'] = pad_w
+
+    return jsonLayer('Pooling', layer_params, get_input_layers(layer_ops), [])
+
+
+def import_inner_product(layer_ops):
+    inner_product_op = next((x for x in layer_ops if x.type in ['Prod', 'MatMul']))
+    layer_params = {}
+
+    if inner_product_op.type == 'MatMul':
+        layer_params['num_output'] = int(inner_product_op.inputs[1].shape[1])
+
+    return jsonLayer('InnerProduct', layer_params, get_input_layers(layer_ops), [])
+
+
+def import_batchnorm(layer_ops):
+    layer_params = {}
+
```

Comment:
Please follow same styles for newlines, I mean add newlines by following some style guidelines